realtime/cli.py

```diff
@@ -19,9 +20,10 @@ def cli():
 )
 @click.option("--endpoint", required=False, help="Endpoint to send the serialized file to.")
 @click.option("--api-key", required=False, help="API key of the sending user")
-def deploy(file_path, endpoint, api_key):
+@click.option("--base-url", required=False, help="Base URL of Adapt endpoint")
```

Comment:
we should only have base_url or endpoint. not both

Reply:
fixed